threema/gateway/e2e.py

```diff
@@ -680,7 +683,8 @@ class DeliveryReceipt(Message):
     Each delivery receipt message confirms the receipt of one
     or multiple regular text messages.
 
-    .. note:: Sending delivery receipts is not officially supported.
```

Comment:
The response I once got from one of your colleagues was it's unsupported. But if it's supported now - fair enough. We had it in there anyway. :)